web/gui/dashboard_info.js

```diff
@@ -3051,77 +3051,77 @@ netdataDashboard.context = {
 
     'ebpf.tcp_functions': {
         title : 'TCP calls',
-        info: 'This chart shows the number of calls for functions <code>tcp_sendmsg</code>, <code>tcp_cleanup_rbuf</code> and <code>tcp_close</code>, whether they return successfully or fail.'
+        info: 'Number of calls for functions <code>tcp_sendmsg</code>, <code>tcp_cleanup_rbuf</code> and <code>tcp_close</code>, whether they return successfully or fail.'
     },
 
     'ebpf.tcp_bandwidth': {
         title : 'TCP bandwidth',
-        info: 'TCP bandwidth shows the number of bytes sent and received for functions <code>tcp_sendmsg</code> and <code>tcp_cleanup_rbuf</code>'
+        info: 'Number of bytes sent and received for functions <code>tcp_sendmsg</code> and <code>tcp_cleanup_rbuf</code>'
     },
 
     'ebpf.tcp_error': {
         title : 'TCP errors',
-        info: 'It shows the number of calls that returned negative values for functions <code>tcp_sendmsg</code> and <code>tcp_cleanup_rbuf</code> and <code>tcp_close</code>.'
+        info: 'Number of calls that returned negative values for functions <code>tcp_sendmsg</code> and <code>tcp_cleanup_rbuf</code> and <code>tcp_close</code>.'
     },
 
     'ebpf.udp_functions': {
         title : 'UDP calls',
-        info: 'UDP functions shows the number of calls for functions <code>udp_sendmsg</code> and <code>udp_recvmsg</code>, whether they return successfully or fail.'
+        info: 'Number of calls for functions <code>udp_sendmsg</code> and <code>udp_recvmsg</code>, whether they return successfully or fail.'
     },
 
     'ebpf.udp_bandwidth': {
         title : 'UDP bandwidth',
-        info: 'UDP bandwidth shows the number of bytes sent and received for functions <code>udp_sendmsg</code> and <code>udp_recvmsg</code>'
+        info: 'Number of bytes sent and received for functions <code>udp_sendmsg</code> and <code>udp_recvmsg</code>'
     },
 
     'ebpf.file_descriptor': {
         title : 'File access',
-        info: 'File descriptor shows the number of calls for internal functions on Linux kernel. The open dimension is attached to the kernel internal function <code>do_sys_open</code>, which is the common function called from open(2) and openat(2). The close dimension is attached to the function <code>__close_fd</code>, which is called from system call close(2).'
+        info: 'Number of calls for internal functions on Linux kernel. The open dimension is attached to the kernel internal function <code>do_sys_open</code>, which is the common function called from open(2) and openat(2). The close dimension is attached to the function <code>__close_fd</code>, which is called from system call close(2).'
```

Comment:
What are those open(2)/close(2)? Probably [a link](https://www.man7.org/linux/man-pages/man2/close.2.html) would help.

Reply:
Thank you very much for the suggestions, I added links for all syscalls.